sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceAsyncClient.java

```diff
@@ -504,7 +504,7 @@ public String getAccountUrl() {
      * @param properties Configures the service.
      * @return A {@link Mono} containing the storage account properties.
      */
-    public Mono<Void> setProperties(BlobServiceProperties properties) {
+    public Mono<Void> setProperties(BlobServiceProperties properties){
```

Comment:
please revert this change

Reply:
So sorry, must have miss-clicked! Fixed now.